tests/integration/security/filebased_tls_origination/egress_gateway_origination_test.go

```diff
@@ -299,10 +301,18 @@ spec:
 `
 )
 
-func createGateway(t test.Failer, ctx resource.Context, appsNamespace namespace.Instance, serviceNamespace namespace.Instance) {
+func createGateway(t test.Failer, ctx resource.Context, appsNamespace namespace.Instance,
+	serviceNamespace namespace.Instance, egressNs string, egressSvc string, egressLabel string,
+) {
 	ctx.ConfigIstio().
-		Eval(appsNamespace.Name(), map[string]string{"ServerNamespace": serviceNamespace.Name()}, Gateway).
-		Eval(appsNamespace.Name(), map[string]string{"ServerNamespace": serviceNamespace.Name()}, VirtualService).
+		Eval(appsNamespace.Name(), map[string]string{
+			"ServerNamespace": serviceNamespace.Name(),
+			"EgressNamespace": egressNs, "EgressLabel": egressLabel, "EgressService": egressSvc,
```

Comment:
nit: was gofmt rub here? This looks off

Reply:
Yeah, this was after running `gofmt` and `gofumpt`.